modules/AOP_Case_Updates/metadata/detailviewdefs.php

```diff
@@ -19,7 +19,7 @@
  * or write to the Free Software Foundation,Inc., 51 Franklin Street,
  * Fifth Floor, Boston, MA 02110-1301  USA
  *
- * @author Salesagility Ltd <support@salesagility.com>
+ * @author SalesAgility Ltd <support@salesagility.com>
```

Comment:
This doesn't match the license in the coding standards

Reply:
Those are files that do not had references to Sugar, so I assumed they was originals from SalesAgility.
I didn't changed the copyright on those as there is no codding standards for non forked files.
Only the brand was fixed.
I think its best to merge it like this and then create a new PR with a different copyright notice if required.